F1WM/ApiModel/Shared/Drivers.cs

```diff
@@ -0,0 +1,12 @@
+using System;
```

Comment:
The model class isn't really "shared" and won't be probably, so the file should land in a different folder.